codecov.yml

```diff
@@ -0,0 +1,22 @@
+codecov:
+  notify:
+    wait_for_ci: true
+  max_report_age: off
+  require_ci_to_pass: true
+comment:
+  behavior: default
+  layout: "reach, diff, flags, files"
+  show_carryforward_flags: false
+coverage:
+  precision: 1
+  range: 97...98 # red -> yellow (the inside range) -> green
```

Comment:
Agreed, so makes sense to have a high upper value, for when status is GREEN.

But if the lower value in the range is below current coverage levels I think this will make all subsequent PRs fail checks, until 97+ is reached. 

But I might be wrong, and if I am right you can always lower it in a coming PR